sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AzureOauth2RefreshTokenGrantRequestEntityConverterTest.java

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.aad.webapp;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.assertj.AssertableWebApplicationContext;
+import org.springframework.boot.test.context.runner.WebApplicationContextRunner;
+import org.springframework.http.RequestEntity;
+import org.springframework.security.oauth2.client.endpoint.OAuth2RefreshTokenGrantRequest;
+import org.springframework.security.oauth2.client.registration.ClientRegistration;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.security.oauth2.core.OAuth2RefreshToken;
+import org.springframework.util.MultiValueMap;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class AzureOauth2RefreshTokenGrantRequestEntityConverterTest {
```

Comment:
After https://github.com/Azure/azure-sdk-for-java/pull/18297 merged, `AzureOauth2RefreshTokenGrantRequestEntityConverterTest.java` can be deleted.